src/backend/distributed/metadata/dependency.c

```diff
@@ -251,6 +273,46 @@ InitObjectAddressCollector(ObjectAddressCollector *collector)
 
 	collector->dependencySet = hash_create("dependency set", 128, &info, hashFlags);
 	collector->dependencyList = NULL;
+
+	collector->visitedObjects = hash_create("visited object set", 128, &info, hashFlags);
+}
+
+
+/*
+ * TargetObjectVisited returns true if the input target has been visited while
+ * traversing pg_depend.
+ */
+static bool
+TargetObjectVisited(ObjectAddressCollector *collector, const ObjectAddress *target)
+{
+	bool found = false;
+
+	/* add to set */
```

Comment:
this comment should probably not be `add to set`, but more like `find in set`.

Guess this might have been copied from the function `IsObjectAddressCollected`. Let's fix there as well

Reply:
Minor: I think we shouldn't have a comment here. The function is already self explanatory. I dont see any value in adding this comment, for me it makes the code dirty.